### PR TITLE
Add my handles pin to the map

### PR DESCRIPTION
**Pull request by mrkadm**

### CODE REVIEW

File: _pins/mrkadm.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+gethubhandle:mrkadm
+latitude:34.792993
+longitude:-92.262494
+---
```
Comment (stoe) [inline _pins/mrkadm.json]: @mrkadm, you have some tiny typos :abcd: in your code.
Please add a space after each colon. Then the test should pass and allow you to merge.

**Comment by mrkadm** [comment]
great, thank you!

**Comment by hollenberry** [comment]
Hey @mrkadm  ! Looks like you've gotten good advice from Stefan above! If you take action on those changes, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@mrkadm I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5090/commits/88a12123641ef8f7e00d2c8282e772695991653f. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**